src/main/java/com/company/Counter.java

```diff
@@ -1,65 +1,36 @@
 package com.company;
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.util.Scanner;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
 
-public class Counter implements WordCounter {
-
-    @Override
-    public int countLines(File file) {
+import java.util.stream.Stream;
 
-        try {
-            Scanner read = new Scanner(file);
-            int numberOfLines = 0;
-            while (read.hasNextLine()) {
-                read.nextLine();
-                numberOfLines++;
-            }
-            return numberOfLines;
-        } catch (FileNotFoundException e) {
-            return -1;
-        }
+@Getter
+@Setter
+@Component
+public class Counter implements WordCounter {
 
-    }
 
     @Override
-    public int countWords(File file) {
-        try {
-            Scanner read = new Scanner(file);
-            int numberOfWords = 0;
-            while (read.hasNextLine()) {
-                numberOfWords += countWordsInOneLine(read.nextLine());
-            }
-            return numberOfWords;
-        } catch (FileNotFoundException e) {
-            return -1;
-        }
-
+    public long countLines(String string) {
+        return Stream.of(string.split("\\n")).count();
     }
 
     @Override
-    public int countCharacters(File file) {
-
-        try {
-            Scanner read = new Scanner(file);
-            int numberOfCharacters = 0;
-            while (read.hasNextLine()) {
-                numberOfCharacters += countCharacterInOneLine(read.nextLine());
-            }
-            return numberOfCharacters;
-        } catch (FileNotFoundException e) {
-            return -1;
-        }
-    }
-
-    private static int countWordsInOneLine(String string) {
-        return string.split("\\s").length;
+    public long countWords(String string) {
+        return Stream.of(string.split("\\s")).count();
```

Comment:
for an input : 
"""
a

a
"""
you will get:
{
    "numberOfLines": 3,
    "numberOfWords": 3,
    "numberOfCharacters": 2
}
Please fix this bug